examples/example.c

```diff
@@ -286,6 +286,10 @@ main(int argc, char **argv)
         sentry_options_set_proxy(options, "socks5://127.0.0.1:1080");
     }
 
+    if (has_arg(argc, argv, "crashpad-wait-for-upload")) {
+        sentry_options_set_on_crash_wait_for_upload(options, true);
```

Comment:
I would rename the option to the same as the argument string. Not for consistency, but because it better highlights what it is for: `sentry_options_set_crashpad_wait_for_upload()` (this also applies to the option flag itself). It only works and makes sense for `crashpad`, which should be highlighted since it is a highly backend-specific function.

Reply:
good catch, thanks!